src/miral/symbols.map

```diff
@@ -470,3 +470,14 @@ global:
   };
 local: *;
 };
+
+MIRAL_5.1 {
+global:
+  extern "C++" {
+    miral::MinimalWindowManager::advise_new_window*;
+    non-virtual?thunk?to?miral::MinimalWindowManager::advise_new_window*;
+    miral::MinimalWindowManager::advise_delete_window*;
+    non-virtual?thunk?to?miral::MinimalWindowManager::advise_delete_window*;
```

Comment:
We don't need a new stanza - we haven't released Miral 5.0 yet. (Just run the regenerate script)